src/Geralt/Interop/Interop.XChaCha20Poly1305.cs

```diff
@@ -8,10 +8,42 @@ internal static partial class Libsodium
         internal const int crypto_aead_xchacha20poly1305_ietf_NPUBBYTES = 24;
         internal const int crypto_aead_xchacha20poly1305_ietf_ABYTES = 16;
 
+        internal const int crypto_secretstream_xchacha20poly1305_ABYTES = 17;
+        internal const int crypto_secretstream_xchacha20poly1305_HEADERBYTES = 24;
+        internal const int crypto_secretstream_xchacha20poly1305_KEYBYTES = 32;
+
+        // Not sure how or whether to handle this
+        // internal const long crypto_secretstream_xchacha20poly1305_MESSAGEBYTES_MAX = big number;
```

Comment:
I don't know that ~256 GB is worth validating for.  But I also don't know how to even interpret the logic going on to determine this value in the libsodium repo.

Reply:
Those commented lines can be removed. libsodium does a check internally, and you can't pass in a message that large, so I'm not checking for ChaCha20-Poly1305 either.